content/articles/print-is-device.md

```diff
@@ -0,0 +1,178 @@
+---
+title: "Print Is a Device"
+date: "2018-04-02"
+type: "post"
+author_name:  "Zarina Mustapha"
+author_url: "https://ctl.columbia.edu/about/team/mustapha/"
+lede: "Printing web content has not fallen out of favor, and because it is very much ingrained in the culture of content consumption, we need to include the printed media in our web development process. In this post, I offer some guidance because web printing is not as straightforward as one might assume."
+poster: "poster-print-a-device.jpg"
+poster_sourceurl: ""
+poster_source: ""
+topics: 
+- User Experience
+tags: [printability, css]
+---
+“Can you teach us how to make sites print right?” a colleague once asked me. The reason websites are not printing right is probably because not much attention is given to what the sites should look like on paper.
+
+Printing has NOT fallen out of favor, regardless of how much we’ve accepted and appreciated digital content. The printed page is still cheap, portable, and convenient. Because print is very much ingrained in the culture of content consumption, we need to include it in our web development process.
+
+In this post, I offer some guidance, based on my experience, because indeed printing from web to paper is not as straightforward as one might assume.
+
+## Print is a device
+
+The printed paper is a portable, static reading device. It is a component of adaptive design alongside desktop and mobile devices, and assistive technology for accessibility. Because of this, it should be given ample consideration in the web development process.
+
+What you see on the screen is not necessarily what you’ll get when you print because the printed version is not a facsimile of the webpage. The printer renders HTML, applies CSS, and then transfers ink onto the output: physical, fixed size pages of paper. Unless specified exclusively for screen use, all defined general CSS rules, even those for responsive CSS, are applied on the printed output. Therefore, you might see some unintended display on paper if the rule in the general CSS is not filtered out. The print stylesheet rules are needed to accommodate the differences between screen and print.
```

Comment:
I might say here that the *browser* renders the HTML and the CSS into a format the printer can then transfer.